tasks/CentOS/main.yml

```diff
@@ -4,18 +4,18 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Set sensu_pkg_version {{ ansible_distribution }} override
```

Comment:
Also it looks like this only being done for CentOS and FreeBSD, is there something special about those hosts or should this eventually be applied to all the main.ymls?